test/unit/helpers/dom/ConsentRequiredForm.js

```diff
@@ -0,0 +1,38 @@
+define(['./Form'], function (Form) {
```

Comment:
is it supposed to add selenium/integration test? (this file is not being used)

Reply:
It is used in the unit test, selenium will be added on core